### ISSUE TITLE
Distiguish mounted/online data

### ISSUE THREAD

**Issue by lighthunt** [whyboris/Video-Hub-App#508]
For ver 3, since one hub my contain combined folders (from different locations, mounted or not), it would be nice to be able to browse/view only online data ...

This may be solved by either (or both):

* toggle "do not show offline videos" (i.e. folders, videos)
* grey-out or mark somehow offline data

**Comment by lighthunt** [comment]
(btw I would label it as feature request appropriately, but probably can't due to access rights)

**Comment by whyboris** [comment]
Offline data already has an indicator 🎉 - though it's faint (see top-left of any image - a yellow circle at 50% opacity)

🤔 I wonder if it's useful to have a button that will hide offline data ... it's _very very easy_ from the code perspective (just add a *pipe* in the template that simply filters out any video files that are offline) ✅ it's just a matter of whether it would be useful.

Please let me know if the indicator is the best solution, or you think a "hide offline data" button would be an additional useful improvement 😁

**Comment by lighthunt** [comment]
> 
> 
> Offline data already has an indicator 🎉 - though it's faint (see top-left of any image - a yellow circle at 50% opacity) 😎

- [ ] 1. Haven't noticed before, but I can see it now, great! Can one configure size/color/opacity somewhere? It's less visible than I'd like (even if hardcoded, good enough for me, I can recompile)
- [ ] 2. Yellow dot is visible only for videos, but not for folders, they should be marked as well (I myself use the folder view often)

> 
> 🤔 I wonder if it's useful to have a button that will hide offline data ... it's _very very easy_ from the code perspective (just add a _pipe_ in the template that simply filters out any video files that are offline) ✅ it's just a matter of whether it would be useful.
> 

Yes, very useful for me, if easy, I'd like that

> Please let me know if the (already-present) indicator is the best solution, or you think a "hide offline data" button would be an additional useful improvement 😁
> 
> Ps - I'm happy to hear feedback about the "not connected" / "offline" indicator -- if you think it could benefit from a visual redesign I'm all ears 🙂

**Comment by whyboris** [comment]
@lighthunt - for now you can manually change the look of the icon with some CSS:
https://github.com/whyboris/Video-Hub-App/blob/f665af80e16c48732197d4dfb6a6ba3722dbabdf/src/app/components/views/time-and-rez.scss#L32

Adding the dot to folders is doable - I just need to add it 😅  - thanks for the reminder!

I'll take a look about adding a button to filter the not-connected. The super-minor worry I have is that someone will toggle the button by accident, and then freak out that many videos are just gone with no explanation. Perhaps if there's some (semi-)prominent message at the top of the gallery that says "hiding non-connected videos" 🤔  -- I'll have to play around with communicating the state somehow. 

I'm open to suggestions 😁

**Comment by lighthunt** [comment]
> The super-minor worry I have is that someone will toggle the button by accident, and then freak out that many videos are just gone with no explanation

TRUE, not so minor ;-) How about adding text in the left filter pane next to folders/files info, like:
19 folder _(10 offline/invisible)_
355 videos _( .... )_

quite easy, consistent (being with filters), and good enough if the (10 offline/invisible) would be in red color or so

**Comment by whyboris** [comment]
I like the idea! -- will play around with the UI 🤝

**Comment by orkomlosh** [comment]
Hey don't forget that now we have the new snackBar we can also pop it up for a few seconds when the user toggled the button to inform the user.

**Comment by whyboris** [comment]
Notification may be a nice touch too. Though I'd like a permanent reminder -- since after they close the app and re-open it, the "hiding offline videos" button will still be toggled 🙈

**Comment by lighthunt** [comment]
The more I think about it, the more I believe the notification or info does not need to be specific to online/offline only.
Anytime I feel I am missing some thumbnails, I just check the filters on the left, scroll up and down to see what do I have to reset.
Perhaps there can just be some generic message somewhere above thumbnails (I don't know snackBar, perhaps), or at the breadcrumbs (directory path) anytime any filters is on, saying something like: Filters applied [reset], or Filtered [reset] or something.
I am missing one-click filter reset anyway (or if there is one, I missed that ;-))
filters can be hidden breadcrumbs not (?)
my humble opinion.

**Comment by lighthunt** [comment]
> 
> 
> Could you clarify what you mean by:
> 
> > "filters can be hidden breadcrumbs not (?)"
> 
> I'm unsure what you mean. I think the only "breadcrumbs" UI I have is the folder navigation when folder view is enabled. Do you mean something different?
 yes, that's what I meant and you are right and I am wrong of course ;-) sorry, please ignore ...
